src/components/Auth/AuthNavigator.js

```diff
@@ -5,8 +5,8 @@ import wrapNavigator, { routeKeys } from 'components/shared/wrapNavigator';
 import Loading from './Loading';
 import Mnemonic from './Mnemonic';
 import PinSetup from './PinSetup';
-import Ready from './Ready';
```

Comment:
Why do we need to drop `Auth/Ready` route here? This screen is important step of signup process: https://github.com/mobius-network/wallet/blob/079e259ce9336956124e7c79b0c1824962d6864e/src/state/sagas/auth/recoveryFinishSaga.js#L31

Reply:
Hmm I didn't mean to do this! Ill get that fixed, my bad!